src/blocks/ImageTextBlock.tsx

```diff
@@ -124,16 +127,31 @@ export const ImageTextBlock: React.FunctionComponent<ImageTextBlockProps> = ({
               __html: paragraph.html,
             }}
           />
-          {show_button && (
-            <ButtonLinkWithMargin
-              href={getStoryblokLinkUrl(button_link)}
-              type={button_type}
-              size="sm"
-              bold
-            >
-              {button_title}
-            </ButtonLinkWithMargin>
-          )}
+          {show_button &&
```

Comment:
Could it be desirable to split these down to `show_regular_button` and `show_branch_button` or something similar? This condition feels a bit weird as it is now.

Reply:
Or perhaps even an enum, `NO_BUTTON, REGULAR_BUTTON, BRANCH_BUTTON` 🙂

Reply:
I  see what you mean, unsure if i agree tho. It kinda makes sense to me to do the same in all blocks.